src/api/procedures/moveFunds.ts

```diff
@@ -73,20 +73,9 @@ export async function prepareMoveFunds(this: Procedure<Params, void>, args: Para
     });
   }
 
-  const [toIsOwned, portfolioBalances] = await Promise.all([
-    toPortfolio.isOwnedBy(),
-    fromPortfolio.getTokenBalances({
-      tokens: items.map(({ token }) => token),
-    }),
-  ]);
-
-  if (!toIsOwned) {
-    throw new PolymeshError({
-      code: ErrorCode.ValidationError,
-      message: 'You must be the owner of both Portfolios',
-    });
-  }
```

Comment:
This is not what was requested

Reply:
the validation you requested is already implemented in line 62

Reply:
Absolutely right. I'm sorry, my bad